Src/CameraData.cs

```diff
@@ -83,6 +87,15 @@ public static void LoadSettings()
             {
                 ws.WriteLine("GlobalBias='0.0'");
                 ws.WriteLine("Camera={");
```

Comment:
It might even be better to make it:
MenuCamera = { .. }
instead of 
Camera = { Name ='MenuCamera' ... }
But whatever is easier...

Reply:
I really like that idea - I'll put that in the backlog! :)